### PR TITLE
WebKit export of https://bugs.webkit.org/show_bug.cgi?id=257092

### PR DESCRIPTION
**Pull request by mdubet**

### CODE REVIEW

File: css/css-fonts/font-palette-non-ident-font-family-ref.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Tests that an empty font family name is handled correctly</title>
```
Comment (drott) [inline css/css-fonts/font-palette-non-ident-font-family-ref.html:5]: This title does not match the original test, I assume you meant "non-ident" here as well?

File: css/css-fonts/font-palette-non-ident-font-family.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Tests that a non-ident font family name is handled correctly</title>
```
Comment (drott) [inline css/css-fonts/font-palette-non-ident-font-family.html:5]: Could you explain what you need in this test that's not covered for example by `css/css-fonts/font-palette-13.html` - is it the space in the name?

Comment (mdubet) [reply css/css-fonts/font-palette-non-ident-font-family.html:5]: Yes, hypen is part of an ident token in CSS syntax but space is not

Comment (drott) [reply css/css-fonts/font-palette-non-ident-font-family.html:5]: But anything that's in quotes is a CSS string token, isn't it? So I got a bit confused by the test title and would perhaps suggest to clarify that.

**Review by drott — COMMENTED** [review]
Thanks for the quick reply.

File: css/css-fonts/font-palette-non-ident-font-family.html (hunk 2 above)
Comment (mdubet) [reply css/css-fonts/font-palette-non-ident-font-family.html:5]: Blink/WebKit have a peculiar behaviour about font-family serialisation (which is also happening with font-family name matching unfortunately) cf https://github.com/w3c/csswg-drafts/issues/5846

File: css/css-fonts/font-palette-non-ident-font-family-ref.html (hunk 1 above)
Comment (nt1m) [reply css/css-fonts/font-palette-non-ident-font-family-ref.html:5]: Fixed it in https://github.com/web-platform-tests/wpt/commit/2d87052e429aafc70c48b5d02b0a6de574f49c4d

**Pull request merged**